tests/test_reader.py

```diff
@@ -189,9 +179,7 @@ def test_read_multinest():
     ns.plot_1d(['x0', 'x1', 'x2', 'x3'])
 
 
-@pytest.mark.xfail('h5py' not in sys.modules,
-                   raises=NameError,
-                   reason="ultranest reading requires `h5py` package")
+@h5py_mark_xfail
```

Comment:
Should this test be simply skipped rather than xfailed?

Reply:
If we don't run it, then we miss 100% coverage -- one could write specific tests which check for failure, but I think this is neater.